scripts/update-codeowners.js

```diff
@@ -21,52 +20,17 @@ async function main() {
     await writeFile([options.definitelyTypedPath, ".github", "CODEOWNERS"].join("/"), `${header}\n\n${entries.join("\n")}\n`, { encoding: "utf-8" });
 }
 
-const token = /** @type {string} */(process.env.GH_TOKEN);
-const gh = new Octokit();
-const reviewers = ["weswigham", "sandersn", "RyanCavanaugh"]
-const now = new Date();
-const branchName = `codeowner-update-${now.getFullYear()}${padNum(now.getMonth())}${padNum(now.getDay())}`;
-const remoteUrl = `https://${token}@github.com/DefinitelyTyped/DefinitelyTyped.git`;
 runSequence([
     ["git", ["checkout", "."]], // reset any changes
 ]);
 
 main().then(() => {
     runSequence([
-        ["git", ["checkout", "-b", branchName]], // create a branch
         ["git", ["add", ".github/CODEOWNERS"]], // Add CODEOWNERS
         ["git", ["commit", "-m", `"Update CODEOWNERS"`]], // Commit all changes
```

Comment:
I reckon we need an emoji here 😎

```suggestion
        ["git", ["commit", "-m", `"🤖 Update CODEOWNERS"`]], // Commit all changes
```

Reply:
Actually, when I saw this, I couldn't help wondering why the double quoting level, then I saw the `shell: true` in `runSequence`.  I'm deeply in the camp of `shell: true` being a universally bad idea, and I see no reason to keep it (and leave this as a simple string...).

Reply:
@elibarzilay I'm going to merge as-is to keep this PR smaller, but I am interested in cleaning up this script some more. Notably, it's based on scripts from the TS infrastructure, so the same fix should apply to all of them.

In this case, I don't understand what the `shell: false` argument should be in order to remove one level of quoting:

`commit -m "Update CODEOWNERS"`

Is the best I can come up with.

Reply:
@sandersn: Well, if it was a `shell: false`, then the argument would not need any extra level of quotes since each member of the list would become one argument regardless of what's in it.  (IOW, if you just change the `shell:`, you'd see `"`s in the commit messages...)

Reply:
In `["commit", "-m", "Update CODEOWNERS"]`, how would `commit` and `-m` be handled? Wouldn't git end up handling them as if they were part of the command line `git 'commit' '-m' 'Update CODOWNERS'`?